src/components/atoms/ApplyButton.vue

```diff
@@ -1,13 +1,62 @@
 <template>
-  <div class="ApplyButton">
-    ApplyButton
-  </div>
+  <!-- TODO: callbackで押した後の処理を実行 -->
+  <button class="ApplyButton">
+    <span
+      v-if="subText"
+      class="ApplyButton_SubText"
+    >
+      {{ subText }}
+    </span>
+    <p class="ApplyButton_Text"><slot /></p>
+  </button>
 </template>
 
 <script lang="ts">
-import { Component, Vue } from 'vue-property-decorator';
+import { Component, Prop, Vue } from 'vue-property-decorator';
 
 @Component
 export default class ApplyButton extends Vue {
+  @Prop({ type: String, required: false }) subText!: string;
```

Comment:
[公式](https://github.com/kaorun343/vue-property-decorator#Prop)を参考にしたけど、書き方は統一したい
他だと、こんな感じかなー
```suggestion
  @Prop({
    type: String,
    required: false,
  })
  subText!: string;
```

Reply:
書き方っていうのは改行とかのことだよね？
そうだね、基本的に改行＋ケツカンマでいいと思う。

あとは基本的にスタイルガイドを参考にしよう。
https://mitsuruog.github.io/javascript-style-guide/#objects--no-new